FunctionSignatureMap.php

```diff
@@ -6074,7 +6078,7 @@
 'kadm5_get_principals' => ['array', 'handle'=>'resource'],
 'kadm5_init_with_password' => ['resource', 'admin_server'=>'string', 'realm'=>'string', 'principal'=>'string', 'password'=>'string'],
 'kadm5_modify_principal' => ['bool', 'handle'=>'resource', 'principal'=>'string', 'options'=>'array'],
-'key' => ['int|string|null', 'array_arg'=>'array'],
+'key' => ['int|string|null', 'array_arg'=>'array|object'],
```

Comment:
I did not know about this, but `key` works. Thankfully, array_key_first only takes arrays.